OPENROUTER_TESTING.md

```
@@ -0,0 +1,70 @@
+# Testing Fairness & Bias with OpenRouter
+
+Use this configuration to set up your API Automated Test for OpenRouter.
+
+## ✅ OpenRouter — Correct Configuration
+
+### 🔹 API Endpoint URL
+```
+https://openrouter.ai/api/v1/chat/completions
+```
+
+### 🔹 Request Body Template
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)